examples/advanced/quantum_volume.ipynb

```diff
@@ -0,0 +1,109 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Running the Quantum Volume Algorithm\n",
+    "This example walks through the steps of running the quantum volume algorithm on square matrices. It is intended to mirror Algorithm 1 of https://arxiv.org/pdf/1811.12926.pdf. In general, we will generate a model circuit, classically compute its Heavy Output Group, then run various samplers (currently, ideal and noisy simulators) to evaluate how often they generate Heavy results."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from examples.advanced import quantum_volume\n",
+    "import numpy as np\n",
+    "from matplotlib import pyplot as plt\n",
+    "from matplotlib.ticker import FormatStrFormatter\n",
+    "\n",
+    "def generate_chart(depths, average_ideals, average_noisies):\n",
+    "    \"\"\"Create a chart that is designed to look as similar as possible to\n",
+    "       Figure 2 in https://arxiv.org/pdf/1811.12926.pdf.\"\"\"\n",
+    "    fig, axs = plt.subplots()\n",
+    "    axs.scatter(depths, average_ideals, marker='+', c='tab:green', label='Ideal')\n",
+    "    axs.scatter([d + .1 for d in depths], average_noisies, c='tab:red', label='Noisy')\n",
+    "    # Line markers for asymptotic ideal heavy output probability and the ideal Heavy Output Generation threshold.\n",
+    "    axs.hlines((1 + np.log(2)) / 2, 0, 100, colors=['tab:green'], label='Asymptotic ideal', linestyles='dashed')\n",
+    "    axs.hlines(2/3, 0, 100, label='HOG threshold', linestyles='dotted')\n",
```

Comment:
use `axs.axhline`. Then you can remove `(0, 100)`. Then you can remove `set_xlim`. Then the code works if you change the depths to include values > 4

Reply:
Awesome, that's super helpful! Done.